src/components/TableColumnHeader.vue

```diff
@@ -7,7 +7,9 @@
     :aria-disabled="ariaDisabled"
     @click="clicked"
   >
-    {{ label }}
+    <span>
```

Comment:
I would have prefered a v-text, but I seem to be the only one to like v-text, so whatever.